src/document-definitions-shell-maker.js

```diff
@@ -31,9 +31,7 @@ function createShell(docDefinitionsString, originalFilename) {
 
   // The test helper environment includes a placeholder string called "%DOC_DEFINITIONS_PLACEHOLDER%" that is to be replaced with the
   // contents of the document definitions
-  var shellString = shellTemplateString.replace(
-    '%DOC_DEFINITIONS_PLACEHOLDER%',
-    function() { return unescapeBackticks(docDefinitionsString); });
+  var shellString = shellTemplateString.replace('%DOC_DEFINITIONS_PLACEHOLDER%', function() { return docDefinitionsString; });
```

Comment:
This is not directly related to the primary purpose of this pull request, but it fixes a subtle mistake.

The document-definitions-shell-maker module previously "unescaped" the character sequence "\\\`" when encountered in a document definitions file. However, that behaviour is only desirable when the test-helper module loads a _sync function_ that has had such escape sequences inserted by the sync-function-loader module. The sync-function-loader module escapes backtick characters so that they do not generate syntax errors because that character is also used to denote multiline strings in Sync Gateway configuration files. Since the document definitions that are passed to document-definitions-shell-maker have not gone through the sync-function-loader, the backticks have not been escaped and any occurrence of the character sequence "\\\`" should be treated literally.